docs/root/faq/api/control_plane.rst

```diff
@@ -0,0 +1,27 @@
+How do I support multiple xDS API major versions in my control plane?
```

Comment:
In a follow up should we actually have a FAQ entry around a control plane doing an upgrade, even if they want to support a single version? There have been tons of questions about this and I wonder if we should provide some ideas/guidance/specifics especially when using go-control-plane? Thoughts? cc @kyessenov @jyotimahapatra @envoyproxy/maintainers @envoyproxy/api-shepherds

Reply:
Yeah, I think this has been quite thoroughly explored in https://github.com/istio/istio/issues/19885. I will make a note to work on a followup; some of the advice will be conditional on items such as https://github.com/envoyproxy/envoy/issues/10776 and https://github.com/envoyproxy/envoy/issues/8421.

Reply:
Im not sure if https://github.com/istio/istio/issues/19885 captures the scenario where the control plane is migrated to v3 and serves v3 responses to v2 requests by just changing the typeurl. 
Since the lds/cds resource versions will be in bootstrap config, it will be very difficult to move our whole fleet to v3 without [this](https://github.com/envoyproxy/envoy/issues/10776) solution. 
I think go-control-plane integration tests can easily demonstrate such scenarios.